bin/_test-helpers.sh

```diff
@@ -6,7 +6,7 @@ set +e
 
 ##### Test setup helpers #####
 
-export default_test_names=(deep external-issuer helm-deep helm-upgrade uninstall upgrade-edge upgrade-stable)
+export default_test_names=(deep external-issuer external-prometheus helm-deep helm-upgrade uninstall upgrade-edge upgrade-stable)
```

Comment:
Given all the `deep` tests are run on this new suite, I think it should be called `external-prometheus-deep` to follow the pattern.